learning/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -4,8 +4,9 @@ Introduction
 ============
 
 .. warning:: C# support is a new feature in Godot 3.0.
-	     As such, you may still run into some issues or find spots where the documentation could be improved.
-	     Please report those issues on `Github <https://github.com/godotengine/godot/issues>`_.
+	     As such, you may still run into some issues, or find spots where the documentation could be improved.
+	     Please report issues with C# in Godot on the `engine Github page <https://github.com/godotengine/godot/issues>`_.
+         And any documentation issues on the `documentation Github Page <https://github.com/godotengine/godot-docs/issues>`_.
```

Comment:
There's an indentation issue here, it should be aligned below `C#` after the `.. warning::`. Note that the previous two lines were badly indented initially, they should use spaces and not a mix of tabs and spaces.